packages/identity-service/sequelize/migrations/20240610205258-add-address-to-auth-table.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.addColumn('Authentications', 'walletAddress', {
```

Comment:
we should add a db index on walletAddress

Reply:
good call